### DISCUSSION TITLE
Paperless LXC Database Location

### DISCUSSION THREAD

**Discussion by daniE43** [tteck/Proxmox#3039] [Help Needed] [answered]
So I set up the Paperless Container and all is well. In order to make some backups, I went to locate all the directories I want backed up. 
media, data, static (probably not needing this one). But I am unable to locate the postgresdb. 

Thanks in advance

**Comment by Mellowlynx** [comment, marked as answer]
PostgreSQL runs as a service, you need to connect to it, before you can make a database dump.
`psql -h localhost -p 5432 -d paperlessdb -U paperless`

The database password can be requested by running `update` from within the Paperless LXC.

**Comment by daniE43** [comment]
Thx for you answer! I guess I was expecting a file nontheless - not that firm with self hosting databases. 
So what would be a practical approach to doing backups? Create some script for the dumps?

**Reply by Mellowlynx** [reply]
It depends on how you want to make the backups.
You can make them from within the LXC and create a script that makes a DB dump, copies the file and compresses everything together before sending it to a NAS or something.

I just back up the full LXC, If I need something it is, but can't override the original one, I restore it somewhere else en get the data that way.

**Reply by daniE43** [reply]
Fullbackup sound more reasonable. Guess I didnt see the forest for the trees...
